apis/python/notebooks/tutorial_soma_append_mode.ipynb

```diff
@@ -673,6 +686,408 @@
         "    print()\n",
         "    print(X.used_shape())"
       ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "tags": []
+      },
+      "source": [
+        "## Ingesting multiple datasets to a SOMA Experiment"
```

Comment:
Please make this an h1 with single `#`

Reply:
I retract this -- you showed me how this is rendered correctly RTD

Reply:
ftr, it currently renders like:

![image](https://github.com/single-cell-data/TileDB-SOMA/assets/465045/45809425-ddfa-4e4c-91b4-d9a618fde6c2)

([link](https://tiledbsoma.readthedocs.io/en/rw-apptut/tutorials.html))

It seems that `h1`s map to the top level of this listing, regardless of notebook boundaries; here's what it looks like with this heading as an `h1`:

![image](https://github.com/single-cell-data/TileDB-SOMA/assets/465045/0f3e4e3d-db51-4d3e-b661-a7fd6ae93af7)